src/service/FileSystemService.java

```diff
@@ -0,0 +1,90 @@
+package service;
+
+import model.Directory;
+import model.File;
+import model.Item;
+import repository.ItemRepository;
+
+
+public class FileSystemService {
```

Comment:
You can have something like IFileSystemService and define all these method definitions there and then create this FileSystemService with the exact method code